src/plotApplications/reducer.js

```diff
@@ -24,10 +24,10 @@ import type {
   ReceiveAttachmentAttributesAction,
   ReceiveAttachmentMethodsAction,
   ReceiveInfoCheckAttributesAction,
-  PlotApplicationsState
-} from "./types";
-import type {Action, CombinedReducer} from "redux";
-import type {ReceiveFormAttributesAction} from "../plotSearch/types";
+  PlotApplicationsState,
+} from './types';
+import type {Action, CombinedReducer} from 'redux';
```

Comment:
You could escape the "untyped-type-import" from this line with `// $FlowFixMe[untyped-type-import]`

There is nothing we can't do about "redux" not having flow types set up...